Src/Microsoft.Dynamic/Actions/ComboBinder.cs

```diff
@@ -242,31 +242,25 @@ public class ParameterMappingInfo {
 
         public override bool Equals(object obj) {
             ParameterMappingInfo pmi = obj as ParameterMappingInfo;
-            if (pmi == null) {
-                return false;
-            }
-
-            if (pmi.ParameterIndex == ParameterIndex && pmi.ActionIndex == ActionIndex) {
+            if (pmi?.ParameterIndex == ParameterIndex && pmi.ActionIndex == ActionIndex) {
```

Comment:
I'm not a fan of using the operator in this type of situation (the possibly null value is used in more than one of the conditions).